src/stats.c

```diff
@@ -249,6 +249,10 @@ void dump_stats(bool log, const char *cache_path) {
     snprintf(str, MAX_LINE_LEN, "  key2path:         %u", FETCH(filecache_key2path));
     print_line(log, fd, LOG_NOTICE, SECTION_FILECACHE_OUTPUT, str);
 
+    snprintf(str, MAX_LINE_LEN, "Notable Events:");
+    print_line(log, fd, LOG_NOTICE, SECTION_FUSEDAV_OUTPUT, str);
```

Comment:
Adding a new "Notable Events" heading here would seemingly affect all following metrics, wouldn't it?  That is reclassifying the latency outputs printed next.

Reply:
Also right, fixed this too by moving it down below latency stuff but above statcache heading.